editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -730,7 +568,7 @@ export const ClassNameSelect = betterReactMemo(
             NoOptionsMessage,
             Menu,
             MultiValueContainer,
-            ValueContainer,
+            // ValueContainer,
```

Comment:
I think this is still neeeded, the ValueContainer had a wrapper div for the css, it was added when we were fixing the sizing with lots of classnames selected the editor became wider than the screen

Reply:
Oops! Good catch, this wasn't supposed to be commented out